custom_components/dualmode_generic/climate.py

```diff
@@ -56,6 +58,7 @@
 
 CONF_HEATER = "heater"
 CONF_COOLER = "cooler"
+CONF_VENTILATOR = "ventilator"
```

Comment:
Mentioned in the corresponding issue, but adding here as well. I hate to be pedantic, but could we refer to this as a fan instead of a ventilator? I'd like to keep consistent terminology with the core climate entity, and since we are using its fan mode here, I think just having a `fan: switch.foo` in the config would be ideal.

Reply:
yea, already renamed. It's now:
```yaml
- platform: dualmode_generic
  name: Bedroom Thermostat
  cooler: input_boolean.air_conditioner
  heater: input_boolean.space_heater
  fan: input_boolean.fan
  fan_behaviour: [heater, cooler, neutral] # --> Not Implemented yet
  target_sensor: sensor.shelly_shht
  target_temp: 20.0
  min_temp: 16
  max_temp: 30
  cold_tolerance: 0.4
  hot_tolerance: 0.4
  min_cycle_duration:
      minutes: 2
```

Reply:
well, I did it already :D